views/_notes.erb

```diff
@@ -0,0 +1,43 @@
+<% if notes.present? %>
```

Comment:
Use `unless notes.empty?`. I'm trying to avoid using ActiveSupport core-exts as much as possible. In the future I may switch from ActiveRecord to `Sequel::Model`.